static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -0,0 +1,172 @@
+  project: Project;
+};
+
+const GroupEventEntry = ({event, entryType, group, project}: GroupEventEntryProps) => {
+  const organization = useOrganization();
+  const entries = event.entries.filter(entry => entry.type === entryType);
+
+  if (!entries.length) {
+    return null;
+  }
+
+  return (
+    <Fragment>
+      {entries.map((entry, index) => (
+        <EventEntry
+          key={index}
+          projectSlug={project.slug}
+          isShare={false}
+          group={group}
+          {...{organization, event, entry}}
+        />
+      ))}
+    </Fragment>
+  );
+};
+
+const GroupEventDetailsContent = ({
+  group,
+  event,
+  project,
+}: GroupEventDetailsContentProps) => {
+  const organization = useOrganization();
+  const location = useLocation();
+  const hasReplay = Boolean(event?.tags?.find(({key}) => key === 'replayId')?.value);
+
+  if (!event) {
+    return (
+      <NotFoundMessage>
+        <h3>{t('Latest event not available')}</h3>
+      </NotFoundMessage>
+    );
+  }
+
+  const eventEntryProps = {group, event, project};
+
+  return (
+    <Fragment>
+      <EventErrors event={event} project={project} isShare={false} />
+      <EventCause
+        project={project}
+        eventId={event.id}
+        group={group}
+        commitRow={CommitRow}
+      />
+      {event.userReport && (
+        <EventDataSection title="User Feedback" type="user-feedback">
+          <EventUserFeedback
+            report={event.userReport}
+            orgId={organization.slug}
+            issueId={group.id}
+          />
+        </EventDataSection>
+      )}
+      <EventTagsAndScreenshot
+        event={event}
+        organization={organization}
+        projectSlug={project.slug}
+        location={location}
+        isShare={false}
+      />
+      <EventEvidence event={event} group={group} />
+      <GroupEventEntry entryType={EntryType.MESSAGE} {...eventEntryProps} />
+      <GroupEventEntry entryType={EntryType.EXCEPTION} {...eventEntryProps} />
+      <GroupEventEntry entryType={EntryType.STACKTRACE} {...eventEntryProps} />
+      <GroupEventEntry entryType={EntryType.THREADS} {...eventEntryProps} />
+      {group.issueCategory === IssueCategory.PERFORMANCE && (
+        <SpanEvidenceSection
+          event={event as EventTransaction}
+          organization={organization}
+        />
+      )}
+      <GroupEventEntry entryType={EntryType.HPKP} {...eventEntryProps} />
+      <GroupEventEntry entryType={EntryType.CSP} {...eventEntryProps} />
+      <GroupEventEntry entryType={EntryType.EXPECTCT} {...eventEntryProps} />
+      <GroupEventEntry entryType={EntryType.EXPECTSTAPLE} {...eventEntryProps} />
+      <GroupEventEntry entryType={EntryType.TEMPLATE} {...eventEntryProps} />
+      <GroupEventEntry entryType={EntryType.BREADCRUMBS} {...eventEntryProps} />
+      <Resources
+        description={getResourceDescription(group.issueType)}
+        links={getResourceLinks(group.issueType, event.platform)}
+      />
```

Comment:
Hmm I think this still does that, no? It's a bit confusing, but the entry types `HPKP, CSP, EXPECTCT, EXPECTSTAPLE, TEMPLATE` are never present in performance events. So it will always render below `BREADCRUMBS` when included, but otherwise it will be directly below the span evidence.

Reply:
🤔 that makes sense, then! cc @0Calories